sirepo/package_data/static/json/schema-common.json

```diff
@@ -83,6 +83,9 @@
         "updateFolder": "/update-folder",
         "uploadFile": "/upload-file/<simulation_type>/<simulation_id>/<file_type>"
     },
+    "commonDynamicModules": [
+        "/static/js/ext/d3-3.5.9.min.js"
```

Comment:
/static/js should not be everywhere. It should be generated on the python side. commonDynamicModules is a funny name. What needs to happen is that schema-common needs to append or prefix the lists of dynamicModules in the python.